models/issue_mail.go

```diff
@@ -70,17 +70,17 @@ func mailIssueCommentToParticipants(e Engine, issue *Issue, doer *User, content
 		if err != nil {
 			return fmt.Errorf("GetUserByID [%d]: %v", watchers[i].UserID, err)
 		}
-		if to.IsOrganization() {
+		if to.IsOrganization() || to.EmailNotifications() == EmailNotificationsEnabled {
```

Comment:
Is this condition really right? Shouldn't it be:
```suggestion
		if to.IsOrganization() || to.EmailNotifications() != EmailNotificationsEnabled {
```

Reply:
Whoops, not sure how I missed that. I've changed it now.